demo/src/screens/componentScreens/SliderScreen.js

```diff
@@ -41,7 +41,7 @@ export default class SliderScreen extends Component {
             <Image assetName={'megaphone'} style={styles.image}/>
             <Slider 
               onValueChange={this.onSliderValueChange} 
-              value={INITIAL_VALUE} 
+              value={this.state.sliderValue} 
```

Comment:
Why do we need to keep updating the value by state here? The 'value' prop is for the **initial** value, once you update it the Slider takes care of the value and uses 'onValueChange' to update the user with the new value. When we update it back to the Slider we'll have an extra render on any value change which can affect performance.